controller/rest/dlp_rule.go

```diff
@@ -1675,16 +1675,30 @@ func handlerDlpExport(w http.ResponseWriter, r *http.Request, ps httprouter.Para
 		resp.Items = append(resp.Items, &resptmp)
 	}
 
-	// tell the browser the returned content should be downloaded
-	var data []byte
 	filename := "cfgDlpExport.yaml"
-	w.Header().Set("Content-Disposition", "Attachment; filename="+filename)
-	w.Header().Set("Content-Encoding", "gzip")
-	w.WriteHeader(http.StatusOK)
+	var data []byte
 	json_data, _ := json.MarshalIndent(resp, "", "  ")
 	data, _ = yaml.JSONToYAML(json_data)
-	data = utils.GzipBytes(data)
-	w.Write(data)
+
+	if isRemoteExport(rconf.ExportOptions) {
+		remoteExport := RemoteExport{
+			DefaultFilePath: filename,
+			Options:         rconf.ExportOptions,
+			Content:         data,
+		}
+		err = remoteExport.Do()
+		if err != nil {
```

Comment:
When err is nil, does it need to return 200?
When err is non-nil, what is returned to the REST API client?

Reply:
Added relevant responses in recent commit.